.editorconfig

```diff
@@ -8,7 +8,7 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 
-[{package.json,.travis.yml}]
+[{package.json,.travis.yml,composer.json}]
```

Comment:
Done. This aligns better with core: https://github.com/WordPress/wordpress-develop/blob/4.7.5/.editorconfig#L16-L18

The `package.json` should be updated to use 2-space indents.